SPEC.md

```diff
@@ -266,14 +266,36 @@ A node with file reference information.
 ### Include
 
 ```js
-interface Include <: BlockNode, FileNode {
+interface IncludeNode <: BlockNode, FileNode {}
```

Comment:
I don't think `RawInclude` should be a `BlockNode`?  Perhaps we should remove `IncludeNode` and just have `Include` and `RawInclude` directly extend `FileNode`.